# HG changeset patch
# User HackaLittleBit
# Date 1491214175 -3600
#      Mon Apr 03 11:09:35 2017 +0100
# Branch trunk
# Node ID 6ad554089dd129f74a9c51e5207feb76cd618b97
# Parent  67f3d08bce63872f69b1300a3ac8e1660d924294
Codechange:Make text clearer for viewport.

diff -r 67f3d08bce63 -r 6ad554089dd1 src/lang/english.txt
--- a/src/lang/english.txt	Mon Mar 27 23:48:22 2017 +0100
+++ b/src/lang/english.txt	Mon Apr 03 11:09:35 2017 +0100
@@ -880,9 +880,9 @@
 
 # Extra view window
 STR_EXTRA_VIEW_PORT_TITLE                                       :{WHITE}Viewport {COMMA}{YELLOW}{NBSP}{RAW_STRING}
-STR_EXTRA_VIEW_MOVE_VIEW_TO_MAIN                                :{BLACK}Copy to viewport
+STR_EXTRA_VIEW_MOVE_VIEW_TO_MAIN                                :{BLACK}Scroll to main
 STR_EXTRA_VIEW_MOVE_VIEW_TO_MAIN_TT                             :{BLACK}Copy the location of the main view to this viewport
-STR_EXTRA_VIEW_MOVE_MAIN_TO_VIEW                                :{BLACK}Paste from viewport
+STR_EXTRA_VIEW_MOVE_MAIN_TO_VIEW                                :{BLACK}Scroll main window
 STR_EXTRA_VIEW_MOVE_MAIN_TO_VIEW_TT                             :{BLACK}Paste the location of this viewport to the main view
 
 # Game options window
